applicatie/logic/controles.py

```diff
@@ -0,0 +1,42 @@
+from jsonschema import Draft4Validator
+import logging
+
+
+_logger = logging.getLogger(__file__)
+
+
+def controle_met_schema(json_bestand, json_schema):
+    """"Controleer een json data bestand
+    aan de hand van een json schema.
+    We gebruiken hierbij het jsonschema package
+    zie https://pypi.org/project/jsonschema/ """
+    foutmeldingen = []
+    v = Draft4Validator(json_schema)
+    errors = sorted(v.iter_errors(instance=json_bestand), key=lambda e: e.path)
```

Comment:
Worden errors automatisch gesorteerd op regelnummer? Waarom sorteer je op path, je controleert toch maar één bestand?

Reply:
Gebruik je deze functie nu zowel voor het data- als het definitiebestand?

Reply:
niet gedaan, laten we zo. Gaan we later bekijken (nu geen prioriteit)